src/CMakeLists.txt

```diff
@@ -1,10 +1,20 @@
-# Copyright (c) 2023 The Bitcoin Core developers
+# Copyright (c) 2024-present The Bitcoin Core developers
 # Distributed under the MIT software license, see the accompanying
-# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+# file COPYING or https://opensource.org/license/mit/.
 
 include(GNUInstallDirs)
 include(AddWindowsResources)
 
+option(SECP256K1_DISABLE_SHARED "" ON)
+option(SECP256K1_ENABLE_MODULE_ECDH "" OFF)
+option(SECP256K1_ENABLE_MODULE_RECOVERY "" ON)
+option(SECP256K1_BUILD_BENCHMARK "" OFF)
+option(SECP256K1_BUILD_TESTS "" OFF)
+option(SECP256K1_BUILD_EXHAUSTIVE_TESTS "" OFF)
+option(SECP256K1_BUILD_CTIME_TESTS "" OFF)
+add_subdirectory(secp256k1)
```

Comment:
Does this need `EXCLUDE_FROM_ALL` to keep it from installing?

Reply:
Nope :)

Thanks to: https://github.com/hebasto/bitcoin/blob/43a66c55ec8770cf7c21112aac9b997f3f2fb704/src/secp256k1/CMakeLists.txt#L52

Reply:
I think it still makes sense to go ahead and set that option though, since this is what it's meant for. We could remove the `SECP256K1_BUILD_*` options and not have to worry about any new binaries being added in the future.

Reply:
Thanks! Reworked.

Reply:
See also https://github.com/bitcoin-core/secp256k1/pull/1545#issuecomment-2182361968.